### PR TITLE
YLP-2348 New columns from glycemia indicators in patients list

### PR DESCRIPTION
**Pull request by celia-rouis**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

File: packages/yourloops/test/integration/pages/home/home.caregiver.spec.tsx [hunk 1]
```diff
@@ -97,34 +102,36 @@ describe('Caregiver home page', () => {
...
-      }
+      },
+      glycemiaIndicators
     })
     jest.spyOn(PatientApi, 'getPatients').mockResolvedValue([patient1, patient2, patient3])
 
     renderPage('/')
 
-    expect(await screen.findByTestId('current-patient-list-grid')).toBeVisible()
+    // expect(await screen.findByTestId('current-patient-list-grid')).toBeVisible()
```
Comment (TimCanu) [inline packages/yourloops/test/integration/pages/home/home.caregiver.spec.tsx:112]: Forgotten commented line

File: packages/yourloops/components/patient-list/current-patient-list/current-patient-list.hook.tsx [hunk 2]
```diff
@@ -196,14 +224,14 @@ export const useCurrentPatientListHook = (props: CurrentPatientListHookProps): P
...
-  const buildPrivateTeamColumns = useCallback((): GridColDef[] => {
+  const buildPrivateTeamOrCaregiverColumns = useCallback((): GridColDef[] => {
     const fieldsNotWanted = [PatientListColumns.Messages, PatientListColumns.MonitoringAlerts]
     return medicalTeamsColumns.filter(column => !fieldsNotWanted.includes(column.field as PatientListColumns))
   }, [medicalTeamsColumns])
 
   const columns: GridColDef[] = useMemo(() => {
-    return user.isUserCaregiver() || selectedTeam.type === TeamType.private ? buildPrivateTeamColumns() : medicalTeamsColumns
-  }, [buildPrivateTeamColumns, medicalTeamsColumns, selectedTeam, user])
+    return user.isUserCaregiver() || selectedTeam.type === TeamType.private ? buildPrivateTeamOrCaregiverColumns() : medicalTeamsColumns
```
[the remark this answers is not in the document]
Comment (celia-rouis) [reply packages/yourloops/components/patient-list/current-patient-list/current-patient-list.hook.tsx:233]: Ticket created: https://diabeloop.atlassian.net/browse/YLP-2403

**Review by TimCanu — APPROVED** [review]

**Pull request merged**